test/WebSites/ModelBindingWebSite/Controllers/VehicleController.cs

```diff
@@ -13,6 +14,21 @@ namespace ModelBindingWebSite
 {
     public class VehicleController : Controller
     {
+        private static Dictionary<int, VehicleViewModel> _vehicles = new Dictionary<int, VehicleViewModel>
+        {
+            {
+                42,
+                new VehicleViewModel
+                {
+                    InspectedDates = new[] { DateTimeOffset.Parse("1 April 2001"), },
```

Comment:
Nevermind. I just wasn't certain if this would've successfully parsed:

```C#
Thread.CurrentThread.CurrentCulture = CultureInfo.GetCultureInfo("fr-FR");
Thread.CurrentThread.CurrentUICulture = CultureInfo.GetCultureInfo("fr-FR");

Console.WriteLine(DateTimeOffset.Parse("1 April 2001"));
```

But it does.